src/ios/classes/AppDelegate+SalesforceHybridSDK.m

```diff
@@ -26,15 +26,15 @@
 #import "AppDelegate+SalesforceHybridSDK.h"
 #import "UIApplication+SalesforceHybridSDK.h"
 #import "InitialViewController.h"
-#import "SFLocalhostSubstitutionCache.h"
-#import "SFHybridViewConfig.h"
-#import "SalesforceSDKManager.h"
-#import "SFUserAccountManager.h"
-#import "SFPushNotificationManager.h"
-#import "SFSDKAppConfig.h"
-#import "SFSDKAuthHelper.h"
-#import "SalesforceHybridSDKManager.h"
-#import "SFSDKHybridLogger.h"
+#import <SalesforceHybridSDK/SFLocalhostSubstitutionCache.h>
```

Comment:
Now that we're purely using dynamic frameworks, we can use proper modular imports.